### PR TITLE
Bug/azure identity binding causing AZ-AAD-AI-1.1 to fail

### PR DESCRIPTION
**Pull request by simonwoolhouse**
Bug has been fixed. CreateAIB successfully creates an Azure Identity Binding programmatically in a non-default namespace as required.
Please review the code

Still WIP as CreateAIB unit test is pending. Test is pretty meaningless without a connection to a test kubernetes cluster

### CODE REVIEW

File: service_packs/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ type PodPayload struct {
 }
 
 func init() {
-	AssetsDir = filepath.Join("service_packs", "kubernetes", "assets")
+	AssetsDir = filepath.Join("../service_packs", "kubernetes", "assets")
```
Comment (eddie-knight) [inline service_packs/kubernetes/kubernetes.go:43]: I believe this is a temporary fix for the VSCode debugger. Won't work for basic functionality, will it?

File: service_packs/service_packs.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ func init() {
 }
 
 func makeGodogProbe(pack string, p probe) *coreengine.GodogProbe {
-	box := utils.BoxStaticFile(pack+p.Name(), "service_packs", pack, p.Name()) // Establish static files for binary build
+	box := utils.BoxStaticFile(pack+p.Name(), "../service_packs", pack, p.Name()) // Establish static files for binary build
```
Comment (eddie-knight) [inline service_packs/service_packs.go:37]: This is a temporary fix specific to the debugger... right?

File: service_packs/kubernetes/kubernetes.go (hunk 1 above)
Comment (simonwoolhouse) [reply service_packs/kubernetes/kubernetes.go:43]: Oops - shouldn't have committed this. Meant to only update locally to allow probes to work in the debugger.
https://github.com/citihub/probr/issues/122 was created for this issue

File: service_packs/service_packs.go (hunk 2 above)
Comment (simonwoolhouse) [reply service_packs/service_packs.go:37]: As per above comment

**Review by eddie-knight — APPROVED** [review]

**Pull request merged**